### ISSUE TITLE
[COMMUNITY] What would you like to see in our Zenith website redesign?

### ISSUE THREAD

**Issue by tenatzen** [finos/zenith#139]
Please add your thoughts and comments on how we can improve the [Zenith Website](https://zenith.finos.org/)

The @finos/zenith-maintainers team are particularly looking for feedback on the following aspects of how we can better report and increase visibility of work within the SIG:

- About Zenith
- Primers
- Brain Trust
- POC Program
- Getting Involved

**Comment by alagergren** [comment]
@tenatzen @@finos/zenith-maintainers

------------------------------
- Comments On "About Zenith" -
------------------------------
TL;DR:  When I first logged on the page I had to really go back and forth between YouTube, FINOS Homepage, Zenith SIG site, and googling random keywords, to understand what exactly Zenith Sig is/does. 

I suggest that upon landing on Zenith Homepage, without scrolling, one has a very clear understanding of what Zenith is/does and how it's related to FINOS.


------------------------------
-- Detailed Recommendations --
------------------------------
** DEFINING THE PROBLEM **
My first-time user experience (FTUE) left me having to dig around the site to understand the relationship between FINOS and Zenith SIG.


** HOMEPAGE RECOMMENDATIONS **
1. Place an "About Zenith" link/tab on the top bar on the homepage. 

This should link to a page which offers:
- an explanation of Zenith Sig as an organization
- an explanation of how Zenith Sig relates to FINOS (with a link about to FINOS mainpage)
- Zenith Sig Mission
- the intro YouTube video (https://youtu.be/lO-TRAQA8GE?si=fUfoaPYuydlSOnSK)

2. Have a brief statement on the Zenith Sig Homepage which states exactly what Zenith Sig is/does, and how it's related to FINOS and what FINOS is/does.

**Comment by alagergren** [comment]
@tenatzen 
@finos/zenith-maintainers

------------------------------
## Comments On Mobile Display
------------------------------
#### Using IPhone 14
Certain text appear disjointed.

![image](https://github.com/finos/zenith/assets/82237290/8978963a-67d1-48d1-801b-c45a1549fb18)

![image](https://github.com/finos/zenith/assets/82237290/d0c6c975-31b8-4974-8725-5f14f3d33a0c)

**Comment by alagergren** [comment]
@tenatzen 
@finos/zenith-maintainers

------------------------------
## Comments On Contact Info
------------------------------
I recommend sharing links:

1. The teams LinkedIn pages
2. The YouTube page
3. Any other contact methods such as Slack, etc.

**Comment by alagergren** [comment]
@tenatzen 
@finos/zenith-maintainers

------------------------------
## Leading with "Call To Action"
------------------------------

Another idea to draw a first time user in would be to "lead with a call to action"

Example of this would be that on the homepage you're greeting with something like:

"Do you have expertise in these Emerging Tech Areas?"
--- List All Areas

"Then you're in the right place"

"Take a look at what we need your help with right now"
---- List open calls to action

**Comment by tenatzen** [comment]
Thanks for the detailed feedback! I'll shoot you a note offline @alagergren to get you access to our web design mural

**Comment by daiyaharsh90** [comment]
Somthing that would have helped me when I stumbled upon Zenith is a search functionality to see if my employer is a member of FINOS/LFX? member pages are filled with logos and not searchable.

**Comment by psmulovics** [comment]
> Somthing that would have helped me when I stumbled upon Zenith is a search functionality to see if my employer is a member of FINOS/LFX? member pages are filled with logos and not searchable.

We did a quick fix for this by adding alt text tags to the logos; we would look into where to add a textual version of the same page. I suppose you are with PayPal, they are not a member - yet.
